index.html

```diff
@@ -10,20 +10,33 @@
 <body>
     <header class="header">
              <a href="#">Gleen</a>
-          
 
-        <nav>
-            <img src="./Images/Union.png" alt="hamburger">
-        
-            <ul class="desktop-links">
-                <li><a href="#Portfolio">Portfolio</a></li>
-                <li><a href="#About">About</a></li>
-                <li><a href="#Contact">Contact</a></li>
-            </ul>
-        </nav>
-    </header>
-
-    <section class="headline">
+             <div class="menu">
+            
+                <div id="myNav" class="overlay">
+              
+                      <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
```

Comment:
- [ ] Kindly ensure that the padding and margin applied to your times' icon matches your chosen Figma template. Lets give our users a good user experience 😄 

| Figma Design | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/179600606-11f24083-accd-4249-b1b9-1ca42e1bcfbb.png)  | ![image](https://user-images.githubusercontent.com/63473584/179600537-592a6b0b-7ce9-4647-8716-5eeda5a165ce.png)  |